README.md

```diff
@@ -44,96 +44,3 @@ conda activate uwtools
 ```
 
 Note that the `uwtools` package's actual name will contain version and build information, e.g. `uwtools-0.1.0-py_0`. The `conda create` command will find and use the most recent [semver](https://semver.org/)-compliant package name given the base name `uwtools`. It could also be explicitly specified as `uwtools=0.1.0=py_0`.
-
-## Development
-
-### Creating a development shell
-
```

Comment:
Maybe these should be kept in since the acceptance criteria says, "Link back from README.md to this page."

Reply:
Yes, good idea, just a line or two letting readers know where the docs are, with a hyperlink to https://unified-workflow.readthedocs.io/en/latest/, would be great.

Reply:
Good call: We probably don't need a lot of structure here -- the existing links to the RTD docs is probably good enough!